x-pack/plugins/observability/public/pages/home/section.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+
+export interface ISection {
+  id: string;
+  title: string;
+  icon: string;
+  description: string;
+  href?: string;
+  target?: '_blank';
+}
+
+const Link = styled(EuiButtonEmpty)`
+  height: auto;
+  &:focus {
+    background-color: transparent;
+  }
+  &:hover {
+    .title {
+      text-decoration: underline;
+    }
+  }
+`;
+
+const Icon = styled(EuiIcon)`
+  color: ${(props) => props.theme.eui.euiIconColors.text};
+`;
+
+export const Section = ({ section }: { section: ISection }) => {
+  const { core } = usePluginContext();
+  const { icon, title, description, href } = section;
+
+  const sectionContent = (
+    <EuiFlexGroup gutterSize="m">
+      <EuiFlexItem grow={false}>
+        <Icon type={icon} size="l" />
```

Comment:
`EuiIcon` already takes a color prop - can you use that?
```suggestion
        <EuiIcon type={icon} size="l" color="default" />
```

Reply:
The problem of this is when it's wrapped in an anchor tag the color changes to blue, no matter the theme. That's why I manually set the color in the Icon styled component.

Reply:
@elastic/eui-design do you have any good suggestions for dealing with block-level content inside of an `EuiLink` like this? Or is this kind of local override the best/only way?

Reply:
It seems you may want to use [EuiCard](https://elastic.github.io/eui/#/display/card) instead. You can pass `href`s directly to EuiCard to make the whole thing clickable. Checkout the "Layout" section on how to get the horizontal layout.

Reply:
Also, since @formgeist built the design he’s usually using EUI components and most likely used cards here. Good bet is to ask what components he was using to build stuff.

Reply:
EuiCard, that's what I was thinking! Thanks @cchaos ... @cauemarcondes is there any reason we can't use that?

Reply:
Thanks, @jasonrhodes, and @cchaos I didn't know about that component. Saved me some line of code 🎉